src/aiovodafone/api.py

```diff
@@ -83,10 +83,12 @@ async def _find_login_url(self) -> str:
         reply_text = await reply.text()
         soup = bs4.BeautifulSoup(reply_text, "html.parser")
         meta_refresh = soup.find("meta", {"http-equiv": "Refresh"})
-        if meta_refresh is not None:
-            meta_content = meta_refresh["content"]
-            reply_url = urllib.parse.parse_qs(meta_content, separator="; ")["URL"][0]
-            redirect_url = urllib.parse.urlparse(reply_url)
+        if type(meta_refresh) is bs4.Tag:
+            meta_content = meta_refresh.get("content")
+            reply_url = urllib.parse.parse_qs(str(meta_content), separator="; ")["URL"][
+                0
+            ]
+            redirect_url = urllib.parse.urlparse(str(reply_url))
```

Comment:
```suggestion
        if isinstance(meta_refresh, bs4.Tag):
            meta_content: str = meta_refresh["content"]
            parsed_qs = urllib.parse.parse_qs(meta_content, separator="; ")
            reply_url: str = parsed_qs["URL"][0]
            redirect_url = urllib.parse.urlparse(reply_url)
```

If `content` is missing, we probably do want it to blow up with KeyError as otherwise its going to blow up later with an error that will be harder to figure out

Reply:
Thx, applied your suggestions with a small change to make `mypy` fully happy ;-)